config.json

```diff
@@ -596,4 +597,4 @@
       }
     }
   ]
-}
+}
```

Comment:
done, this always creeps back out when making changes in theme editor, maybe we can adjust stencil cli to add a nl here...